add_to_app/MultipleFluttersIos/MultipleFluttersIos/SingleFlutterViewController.swift

```diff
@@ -0,0 +1,47 @@
+import Foundation
+import Flutter
+
+class SingleFlutterViewController : FlutterViewController, DataModelObserver {
+  var _channel : FlutterMethodChannel?
+  var _engine : FlutterEngine!
+  
+  init(withEntrypoint entryPoint: String?) {
+    let appDelegate : AppDelegate = UIApplication.shared.delegate as! AppDelegate
+    _engine = appDelegate.engines.makeEngine(withEntrypoint: entryPoint, libraryURI: nil)
+    super.init(engine: _engine, nibName: nil, bundle: nil)
+    DataModel.shared.addObserver(observer: self);
+  }
+  
+  deinit {
+    DataModel.shared.removeObserver(observer: self)
+  }
+  
+  required init(coder aDecoder: NSCoder) {
+    fatalError("init(coder:) has not been implemented")
+  }
+  
+  func onCountUpdate(newCount: Int64) {
+    if let channel = _channel {
+      channel.invokeMethod("setCount", arguments: newCount);
+    }
+  }
+  
+  override func viewDidLoad() {
+    super.viewDidLoad()
+    _channel = FlutterMethodChannel(name: "multiple-flutters", binaryMessenger: self.engine!.binaryMessenger)
+    _channel!.invokeMethod("setCount", arguments: DataModel.shared.count)
```

Comment:
why not do these 2 steps in the init too?

Reply:
This is slightly more efficient, doing all the flutter bindings only when we are actually going to view something.  Subscribing to the datamodel isn't as heavy but it could be moved here too.  Updates are ignored anyways until we have a view.

Reply:
SG